code-studio/build-css.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+/** @file Build script for CSS assets in the code-studio package, which is loaded
+    by dashboard (our "Code Studio" Rails app). */
+'use strict';
+
+var build_commands = require('./build-commands');
+var program = require('commander');
+
+/** @const {string} */
+var SRC_PATH = './src/css/';
+
+/** @const {string} */
+var BUILD_PATH = './build/css/';
+
+/**
+ * Paths to search when evaluating scss @import directives.
+ * Rooted at working directory, with NO trailing slash.
+ * @type {string[]}
+ */
+var INCLUDE_PATHS = [
+  'node_modules',
+  'src/css',
+  '../shared/css'
```

Comment:
I'm leaving this ugly, for now; our shared package is technically rooted at shared/js, but I needed to pull in shared/css/colors.scss, and I wasn't sure what approach we want to take.